<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PruneOptionsPage</class>
  <widget class="QWidget" name="pruningToolBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>371</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Prune Options and Archive Naming</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pruning removes older archives. You can choose the number of hourly, daily, etc. archives to preserve. Usually you will keep more newer and fewer old archives. Read &lt;a href=&quot;https://borgbackup.readthedocs.io/en/stable/usage/prune.html&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#FF4500;&quot;&gt;more&lt;/span&gt;&lt;/a&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Hourly:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="prune_hour">
              <property name="toolTip">
               <string>Use -1 for unlimited</string>
              </property>
              <property name="minimum">
               <number>-1</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Daily:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="prune_day">
              <property name="toolTip">
               <string>Use -1 for unlimited</string>
              </property>
              <property name="minimum">
               <number>-1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Weekly:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="prune_week">
              <property name="toolTip">
               <string>Use -1 for unlimited</string>
              </property>
              <property name="minimum">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Monthly:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="prune_month">
              <property name="toolTip">
               <string>Use -1 for unlimited</string>
              </property>
              <property name="minimum">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Annual:</string>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="prune_year">
              <property name="toolTip">
               <string>Use -1 for unlimited</string>
              </property>
              <property name="minimum">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>No matter what, keep all archives of the last:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="prune_keep_within">
              <property name="toolTip">
               <string>“&lt;int&gt;&lt;char&gt;”, where char is “H”, “d”, “w”, “m”, “y”</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>24H, 1d, 52w, 12m, 1y</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="verticalSpacing">
             <number>4</number>
            </property>
            <item row="0" column="1">
             <widget class="QLineEdit" name="prunePrefixTemplate">
              <property name="toolTip">
               <string>Available variables: hostname, fqdn, profile_id, profile_slug, now, utc_now, user</string>
              </property>
              <property name="placeholderText">
               <string>{hostname}-{profile_slug}-</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Prune Prefix:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="prunePrefixPreview">
              <property name="font">
               <font>
                <italic>true</italic>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: grey;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="verticalSpacing">
             <number>4</number>
            </property>
            <item row="0" column="1">
             <widget class="QLineEdit" name="archiveNameTemplate">
              <property name="toolTip">
               <string>Available variables: hostname, fqdn, profile_id, profile_slug, now, utc_now, user</string>
              </property>
              <property name="placeholderText">
               <string>{hostname}-{profile_slug}-{now:%Y-%m-%d-%H%M%S}</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Archive Name:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="archiveNamePreview">
              <property name="font">
               <font>
                <italic>true</italic>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: grey</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    <resources/>
 <connections/>
</ui>
